ansible/roles/install_sphinx/tasks/main.yml

```diff
@@ -21,10 +21,3 @@
   with_items:
     - /etc/init.d/sphinxsearch
     - /etc/default/sphinxsearch
-
-- name: Start and Enable sphinxsearch
```

Comment:
Removing this is not enough. By default on Debian services are started after install.

You either need to mask the (non-existing) service, install, unmask, or return 101 from [policy-rc.d](https://people.debian.org/~hmh/invokerc.d-policyrc.d-specification.txt).

I haven't checked what we do exactly in imouto (if we make sure services are started/enabled after install), maybe the policy-rc.d change would break too many assumptions, maybe not.